### PR TITLE
Candidate parameters make consent required 

### PR DESCRIPTION
**Pull request by ridz1208**
redmine 10264

### CODE REVIEW

**Comment by stellarxo** [comment]
issue persists - field is still not required, can save date without entering it

**Comment by ridz1208** [comment]
@stellarxo did you remove the override ?

**Comment by stellarxo** [comment]
@ridz1208 hahahahaaaa

**Comment by stellarxo** [comment]
image

File: modules/candidate_parameters/php/NDB_Form_candidate_parameters.class.inc [hunk 1]
```diff
@@ -804,7 +805,14 @@ class NDB_Form_candidate_parameters extends NDB_Form
                 $consent           = date_create();
                 $today             = date('Y-m-d'); 
                 $consent->setDate($consentDate['Y'], $consentDate['M'], $consentDate['d']);
-                $consent = $consent->format('Y-m-d'); 
+                $consent = $consent->format('Y-m-d');
+
+                //print_r($consentVal);
```
Comment (stellarxo) [inline modules/candidate_parameters/php/NDB_Form_candidate_parameters.class.inc]: Remove this

File: modules/candidate_parameters/php/NDB_Form_candidate_parameters.class.inc [hunk 2]
```diff
@@ -943,7 +943,7 @@ class NDB_Form_candidate_parameters extends NDB_Form
                     || empty($consentDate['M'])
                     || empty($consentDate['d'])                    )
                 ) {
-                    $errors[$fieldname . '_date'] = 'Required';
+                    $errors[$fieldname . '_date'] = "Date required when YES <br>";
```
Comment (stellarxo) [inline modules/candidate_parameters/php/NDB_Form_candidate_parameters.class.inc]: Update html in template

**Pull request closed without merging**